CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## 0.13.0 / 2016-09-26
```

Comment:
We switched to including `v` in version identifiers. Until `v0.13.0` is released this should not include a date, e.g. `v0.13.0 / unreleased`.